config/base/scene/simple_desktop.json

```diff
@@ -2,17 +2,21 @@
   "startDate": "today",
   "minDate": "1950-01-02 00:00:00.000",
   "maxDate": "2049-12-31 00:00:00.000",
-  "fileLogLevel": "debug",
-  "consoleLogLevel": "trace",
-  "screenLogLevel": "info",
   "observer": {
     "center": "Earth",
     "frame": "IAU_Earth",
-    "distance": 10000000.0,
-    "longitude": 11.281067,
-    "latitude": 48.086709
+    "position": [
+      5915095.557596171,
+      10658059.249942748,
+      15601636.827477392
+    ],
```

Comment:
That's true, but I think it won't be necessary to edit these frequently. You can now just navigate to some place and save the settings. Programmatically it's much easier to use cartesian coordinates instead of converting it always back and forth...